### ISSUE TITLE
You have a rival

### ISSUE THREAD

**Issue by analytic-bias** [mawerty/Is-Prime#206]
Behold, https://store.steampowered.com/app/2696480/Nothing/

Source: https://steamcommunity.com/profiles/76561198014552003/recommended/2696480/

**Comment by mawerty** [comment]
Wdym, this guy made a game, and we implemented the one of the most complex algorithms known to humanity in about 150 languages!

**Comment by analytic-bias** [comment]
No; the most complex algorithm known to humanity, up to a fixed cardinal $\kappa$, is
```haskell
absurd :: a
absurd = absurd
```
since **_ex ea sequitur quodlibet_**.

----

![image](https://github.com/mawerty/Is-Prime/assets/6694864/b1f093e8-20a3-4e01-a279-339d30ae7aa0)

**Comment by analytic-bias** [comment]
I'm going to put this reply into a separate issue (lol
